auto-filter-tests-by-letter/filter_tests_by_letter.py

```diff
@@ -0,0 +1,69 @@
+"""
+This script filters rows from a CSV file based on specific criteria.
+
+It identifies rows where the project name (extracted from a URL column) 
+starts with a given letter and where the "Status" column is empty. 
+The filtered rows are saved to a new CSV file.
+
+Usage:
+    python3 filter_tests_by_letter.py <letter> <file_path>
+
+Arguments:
+    letter: The starting letter of the project name to filter.
+    file_path: The path to the input CSV file.
+"""
+
+import csv
+import argparse
+import sys
+
+
+def filter_tests_by_letter(file_path: str, letter: str) -> tuple[list[str], list]:
```

Comment:
This works only on python 3.9+
python 3.8 would give 
_TypeError: 'type' object is not subscriptable_

You must either use this :
```
from typing import List, Tuple


def filter_tests_by_letter(file_path: str, letter: str) -> Tuple[List[str], list]:
```
or change the requirements in README to Python 3.9 or later.


Other than that the script works fine.

Reply:
@Ajitesh27 Hey Ajitesh, thanks for pointing this out, I've updated the README to 3.9.